cantor-archive/src/main/java/com/salesforce/cantor/archive/SetsArchiverOnFile.java

```diff
@@ -67,4 +68,19 @@ public static void restore(final Sets sets, final String namespace, final Path a
             logger.info("restored {} entries int namespace '{}' from archive file {}", total, namespace, archiveFile);
         }
     }
+
+    @Override
+    public boolean hasArchives(final String namespace, final String set) {
+        return false;
+    }
+
+    @Override
+    public void archive(final Sets sets, final String namespace) throws IOException {
+
```

Comment:
Wherever the implementation is missing, add `//TODO implement this`

Reply:
So this is actually implemented, at least mostly, but this PR doesn't have that work.